packages/ckeditor5-image/docs/_snippets/features/image-resize-px.html

```diff
@@ -6,9 +6,9 @@ <h3>Resize me!</h3>
 		<figcaption>Autumn fields by Aleksander Nowodziński</figcaption>
 	</figure>
 
-	<h3>Resized image (width: 400px):</h3>
```

Comment:
Why do you change this width?

Reply:
Because I wanted the image to be slightly bigger than 400px. I'll bring back the previous value.